README.md

```diff
@@ -33,11 +33,31 @@
 
 ## Why ng2-materialize?
 
-We are aware that alternatives exposing Material Design components already exist. But, in our case, we want to rely on a stable css framework that does not depend on a specific js framework.
+The main purpose of this library is to simplify the usage of Materialize within the Angular framework which make it more dynamic. To illustrate this, we have Playground sections in several component demo pages.
 
-We already know the possibilities Materialize offers. What is left to do is to wrap components that we want to use in Angular component.
+### Initialization simplified
```

Comment:
I think its a bit heavy for a read-me file (TLDR) ... what do you think about only mentioning those with bullet points? Something like ...

> In other words, we ...
> - Simplified components initialization
> - Handle Angular "quirks" with Materialize library
> - Offer component injection when possible
> - Provide a MediaService that allow customization according to view port size
> - Added support for ReactiveForm with validation

Reply:
Pretty good and its a good resume to what we try to do.